src/components/view.js

```diff
@@ -84,6 +107,59 @@ function createPropertyItem(property) {
   li.appendChild(div);
   div.appendChild(priceElement);
   div.appendChild(locationElement);
+  li.addEventListener('click', e => onPropertyClick(e, isFav));
 
   propertyList.appendChild(li);
 }
+
+export function renderPage(page = 'search') {
+  switch (page) {
+    case 'search':
+      searchContainer.style.display = 'flex';
+      propertyContainer.style.display = 'none';
+      propertyPageContainer.style.display = 'none';
+      break;
+    case 'propertiesList':
+      searchContainer.style.display = 'none';
+      propertyContainer.style.display = 'flex';
+      propertyPageContainer.style.display = 'none';
+      break;
+    case 'propertyDetails':
+      searchContainer.style.display = 'none';
+      propertyContainer.style.display = 'none';
+      propertyPageContainer.style.display = 'flex';
+      break;
+    case 'faves':
+      searchContainer.style.display = 'none';
+      propertyContainer.style.display = 'none';
+      propertyPageContainer.style.display = 'none';
+      break;
+    default:
+      searchContainer.style.display = 'flex';
+      propertyContainer.style.display = 'none';
+      propertyPageContainer.style.display = 'none';
+      break;
+  }
+}
+
+export function renderPropertyDetails() {
+  state.favoriteProperties.find(prop => state.currentProperty.property_id === prop.property_id)
+    ? (addPropertyToFavsButton.style.visibility = 'hidden')
+    : (addPropertyToFavsButton.style.visibility = 'visible');
+
+  emptyFavListMsg.style.display = 'none';
+
+  propertySummary.innerHTML =
+    'Lorem ipsum dolor sit amet consectetur adipisicing elit. Rerum blanditiis ducimus necessitatibus iste suntofficia hic sapiente iure eius porro.';
+  propertyFurniture.innerHTML = `${state.currentProperty.beds} ${state.currentProperty.beds <= 1 ? 'bed' : 'beds'}, ${
```

Comment:
Create variables for:
${state.currentProperty.beds <= 1 ? 'bed' : 'beds'}
`${state.currentProperty.baths <= 1 ? 'bath' : 'baths'}`